.env.development

```diff
@@ -24,8 +24,8 @@ LOGO_TRADEMARK_URL=https://edx-cdn.org/v3/default/logo-trademark.svg
 LOGO_WHITE_URL=https://edx-cdn.org/v3/default/logo-white.svg
 FAVICON_URL=https://edx-cdn.org/v3/default/favicon.ico
 COLLECT_YEAR_OF_BIRTH=true
-APP_ID=''
-MFE_CONFIG_API_URL=''
+APP_ID='profile'
```

Comment:
Note: not explicitly necessary unless you opt to define `PARAGON_THEME_URLS` within the MFE configuration API in `edx-platform`.